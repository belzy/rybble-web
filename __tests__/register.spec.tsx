import { render } from '@testing-library/react';
import RegisterPage from '../pages/register';
import '@testing-library/jest-dom';

describe('pages/register', () => {

  it('should render unchanged', () => {

    const { container } = render(<RegisterPage />);

    expect(container).toMatchSnapshot();

  });

});