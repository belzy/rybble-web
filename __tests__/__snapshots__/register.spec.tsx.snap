// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/register should render unchanged 1`] = `
<div>
  <div
    class="css-qy22cp"
  >
    <div
      class="css-10u5hex"
    >
      <div
        class="css-12iz0lu"
      >
        <img
          class="chakra-image css-6qqipc"
          src="/assets/img/rybble-logo32x32.svg"
        />
        <div
          class="carousel carousel slide"
        >
          <div
            class="carousel-indicators"
          >
            <button
              aria-current="true"
              aria-label="Slide 1"
              class="active"
              data-bs-target=""
              type="button"
            />
            <button
              aria-current="false"
              aria-label="Slide 2"
              data-bs-target=""
              type="button"
            />
            <button
              aria-current="false"
              aria-label="Slide 3"
              data-bs-target=""
              type="button"
            />
          </div>
          <div
            class="carousel-inner"
          >
            <div
              class="active carousel-item"
            >
              <h3
                class="chakra-text css-1tz2i1u"
              >
                Get More Done
              </h3>
              <p
                class="chakra-text css-504jus"
              >
                Stay organised, get notified when work happens
                <br />
                and get results
              </p>
            </div>
            <div
              class="carousel-item"
            >
              <h3
                class="chakra-text css-1tz2i1u"
              >
                Stay Connected
              </h3>
              <p
                class="chakra-text css-504jus"
              >
                Stay organised, get notified when work happens
                <br />
                and get results
              </p>
            </div>
            <div
              class="carousel-item"
            >
              <h3
                class="chakra-text css-1tz2i1u"
              >
                Live Smarter
              </h3>
              <p
                class="chakra-text css-504jus"
              >
                Each minute is a little thing, and yet,  to manage
                <br />
                is the secret of success.
              </p>
            </div>
          </div>
          <a
            class="carousel-control-prev"
            href="#"
            role="button"
            tabindex="0"
          >
            <span
              aria-hidden="true"
              class="carousel-control-prev-icon"
            />
            <span
              class="visually-hidden"
            >
              Previous
            </span>
          </a>
          <a
            class="carousel-control-next"
            href="#"
            role="button"
            tabindex="0"
          >
            <span
              aria-hidden="true"
              class="carousel-control-next-icon"
            />
            <span
              class="visually-hidden"
            >
              Next
            </span>
          </a>
        </div>
      </div>
      <img
        class="chakra-image css-ib9iq6"
        src="/assets/img/auth-page-hero560x400.png"
      />
      <div
        class="css-78bdv5"
      >
        <a
          class="chakra-link link css-x0sdsu"
        >
          <h2
            class="chakra-heading css-0"
          >
            PRIVACY POLICY
          </h2>
        </a>
        <a
          class="chakra-link link css-n8m5js"
        >
          <h2
            class="chakra-heading css-0"
          >
            TERMS OF SERVICE
          </h2>
        </a>
      </div>
    </div>
    <div
      class="css-1mcnvz7"
    />
  </div>
</div>
`;
